### DISCUSSION TITLE
Missing left PCB?

### DISCUSSION THREAD

**Discussion by dospes** [snsten/Klein#22] [General]
Hello everyone,
I feel like I am missing something very obvious, but when cloning the repo and looking at the PCB files, I can only find the right PCB, not the left one. Are both part of the same file, and I need to switch between them somehow? Any help would be greatly appreciated.
Cheers!

Edit: I think I am starting to understand, that the PCB can be flipped to be used on either side. If that is the case, the design was so clever that I missed it.

**Comment by snsten** [comment]
Hi,
Yes, the PCB's are reversible. I don't think it's obvious :smiley:, unless you already have experience with it.
